### PR TITLE
[tflchef] SVDF support for network generation

### PR DESCRIPTION
**Pull request by BalyshevArtem**
Add network generation support for SVDF operator

issue: #8203 

ONE-DCO-1.0-Signed-off-by: Artem Balyshev a.balyshev@partner.samsung.com

### CODE REVIEW

**Comment by seanshpark** [comment]
Can you rebase this PR after `SVDF_001` recipe lands?

reason: tflchef codes will be tested in the CI when related recipes exist in code base.
adding tflehef test is one way but will be duplicate so we choosed this order for testing codes.

File: compiler/tflchef/tflite/src/Op/SVDF.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+namespace tflchef
+{
+
+void TFliteOpSVDF::filler(const tflite::Operator *op, TFliteImport *import,
+                          tflchef::ModelRecipe *model_recipe) const
+{
+  const std::vector<int32_t> &inputs = as_index_vector(op->inputs());
+
+  const bool hasBias = (inputs.size() == 5);
+  assert(inputs.size() == 4 || hasBias);
```
Comment (binarman) [inline compiler/tflchef/tflite/src/Op/SVDF.cpp:30]: Do you plan to support optional bias in importer?

Comment (BalyshevArtem) [reply compiler/tflchef/tflite/src/Op/SVDF.cpp:30]: Yes, in this draft #8279 optional bias is supported (you can see it in compiler/luci/import/src/Nodes/CircleSVDF.cpp)

**Comment by BalyshevArtem** [comment]
I have rebased this PR. After this PR will be landed, then we can merge PR with recipes  #8285

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by binarman — APPROVED** [review]
LGTM

**Pull request merged**